src/common/ExportSettingsModal/utils/createExportReport.js

```diff
@@ -132,6 +136,7 @@ const getOrderLineExportData = ({
   locationMap,
   materialTypeMap,
   poLinesMap,
+  userMap,
```

Comment:
I don't think it includes info from order line, look at my fixes for invoice and receiving

Reply:
I am extending it